symengine/fields.cpp

```diff
@@ -434,6 +457,50 @@ GaloisFieldDict::gf_frobenius_map(const GaloisFieldDict &g,
     return out;
 }
 
+std::pair<GaloisFieldDict, GaloisFieldDict> GaloisFieldDict::gf_trace_map(
+    const GaloisFieldDict &a, const GaloisFieldDict &b,
+    const GaloisFieldDict &c, const integer_class &n) const
+{
+    auto n_val = mp_get_si(n);
```

Comment:
We have to change the value of `n`, so we have to use another variable and copy its content. I thought it would be faster on `int` than `integer_class` object, furthermore `&` operator doesn't work on `integer_class`.